src/proxy.c

```diff
@@ -1606,7 +1606,9 @@ cc_proxy_hyper_exec_command (struct cc_oci_config *config)
 		json_array_add_object_element (envs, env_var);
 	}
 
-	json_object_set_string_member (process_node, "workdir", process->cwd);
+	if (process->cwd[0]) {
```

Comment:
@dlespiau `process` is pointing at `config->oci.process` and cwd is defined as follow:

```
struct oci_cfg_process {
	gchar              **args;

	/** Full path to working directory to run workload command in. */
	gchar                cwd[PATH_MAX];
[...]
```

So it can't be NULL if `process` is not NULL. So if a check is needed here, it probably should be on `process` itself, which is a separate issue.